backend/packages/Upgrade/src/api/controllers/FeatureFlagController.ts

```diff
@@ -160,6 +161,54 @@ export class FeatureFlagsController {
     return this.featureFlagService.find(request.logger);
   }
 
+  /**
+   * @swagger
+   * /flags:
+   *    post:
+   *       description: Get feature flags for user
+   *       consumes:
+   *         - application/json
+   *       parameters:
+   *         - in: body
+   *           name: user
+   *           required: true
+   *           schema:
+   *             type: object
+   *             properties:
+   *               userId:
+   *                 type: string
+   *                 example: user1
+   *               context:
+   *                 type: string
+   *                 example: add
+   *             description: User Document
+   *       tags:
+   *         - Feature Flags
+   *       produces:
+   *         - application/json
+   *       responses:
+   *          '200':
+   *            description: Feature Flag List
+   *            schema:
+   *              type: array
+   *              items:
+   *                $ref: '#/definitions/FeatureFlag'
+   *          '401':
+   *            description: AuthorizationRequiredError
+   */
+
+  @Post('/keys')
```

Comment:
In the issue the endpoint was called `featureFlags` rather than `keys`. Do we want it to be `keys`?

Reply:
Post() for feature-flag is already in use for creating feature-flag, therefore I created it `keys` that make the most sense.

Reply:
/featureFlag will be just Post() which is used for creating single feature-flags.
So for now it I made `keys` which make more sense.
We can change it if wanted.

Reply:
we removed this part.